client-app/src/pages/account/components/invoices-filter/index.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div v-if="invoicesFilter && availableInvoicesStatuses">
+    <div class="d-flex flex-wrap flex-sm-row flex-column">
+      <div class="col col-sm-6 col-md-5 col-lg-4">
+        <label for="begin-date">{{ $t("account.invoices.date-filter.from") }}</label>
+        <b-form-datepicker
+          id="begin-date"
+          :value="invoicesFilter.startDate"
+          value-as-date
+          reset-button
+          :label-reset-button="$t('account.invoices.date-filter.reset')"
+          :date-format-options="{ year: 'numeric', month: 'long', day: 'numeric' }"
+          :max="new Date()"
+          :class="{ 'is-invalid': !isDateValid && isDateValid != null }"
+          :locale="locale"
+          v-bind="datepickerLabels"
+          class="mb-2"
+          @input="changeStartDate($event)"></b-form-datepicker>
+      </div>
+      <div class="col col-sm-6 col-md-5 col-lg-4">
+        <label for="end-date">{{ $t("account.invoices.date-filter.to") }}</label>
+        <b-form-datepicker
+          id="end-date"
+          :value="invoicesFilter.endDate"
+          value-as-date
+          reset-button
+          :label-reset-button="$t('account.invoices.date-filter.reset')"
+          :date-format-options="{ year: 'numeric', month: 'long', day: 'numeric' }"
+          :max="new Date()"
+          :class="{ 'is-invalid': !isDateValid && isDateValid != null }"
+          :locale="locale"
+          v-bind="datepickerLabels"
+          class="mb-2"
+          @input="changeEndDate($event)"></b-form-datepicker>
+      </div>
+      <div class="d-flex flex-column justify-content-center col-sm-3 col-md-2 col-lg-2">
+        <label for="dropdown-filters">{{ $t("account.invoices.status-filter.title") }}</label>
+        <b-dropdown
+          id="dropdown-filters"
+          class="mb-2"
+          variant="outline-primary"
+          toggle-class="text-left"
+          menu-class="p-2"
+          no-caret
+          @hide="toggleStatusDropdown()"
+          @show="toggleStatusDropdown()">
+          <template v-slot:button-content>
+            <div class="d-flex justify-content-between align-items-center">
+              {{ getCurrentStatusLabel() }}
+              <font-awesome-layers v-if="showStatusDropdown">
+                <font-awesome-icon :icon="faAngleUp" size="lg"></font-awesome-icon>
+              </font-awesome-layers>
+              <font-awesome-layers v-if="!showStatusDropdown">
+                <font-awesome-icon :icon="faAngleDown" size="lg"></font-awesome-icon>
+              </font-awesome-layers>
```

Comment:
As I can understand, bootstrap-vue's dropdown has angles by default. Why we need it?

Reply:
by default, dropdown caret is too close to label. Therefore, for customization, I used FA with v-slot for button content
![image](https://user-images.githubusercontent.com/22875828/77051842-9743af00-69d4-11ea-8e4b-9fcc91d722fc.png)